dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_h2.sql

```diff
@@ -2183,4 +2183,4 @@ CREATE TABLE t_ds_listener_event
     PRIMARY KEY (id),
     KEY         idx_status (post_status),
     KEY         idx_event_sign (sign)
-);
+);
```

Comment:
Is this necessary? This may be related to one of the CI error

Reply:
Recover this file